### PR TITLE
Fix to double click for new comment

### PR DESCRIPTION
**Pull request by gabrielpoca**
The create comment button needs to be disabled during the sever request in
order to prevent someone from creating the the same comment multiple times.

### CODE REVIEW

**Comment by gabrielpoca** [comment]
@joaojusto please review this

File: app/assets/javascripts/app/components/button/button.js [hunk 1]
```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```
Comment (joaojusto) [inline app/assets/javascripts/app/components/button/button.js]: Shouldn't you bind to controller so we can avoid using the scope? We are doing this on the other directives as well.

Comment (gabrielpoca) [inline app/assets/javascripts/app/components/button/button.js]: I'm not really doing anything with them, just passing to the html, I think it keeps it simple

Comment (joaojusto) [inline app/assets/javascripts/app/components/button/button.js]: I know, I said just to keep the consistency but in this case if you think it's ok then :+1:

Comment (gabrielpoca) [inline app/assets/javascripts/app/components/button/button.js]: I was looking into the other components and I think we can open an exception for this case, but let's get a third opinion on this, @azevedo-252 can you?

Comment (azevedo-252) [inline app/assets/javascripts/app/components/button/button.js]: I'm not a big fan of exceptions, it makes sense now for the moment, for the people looking at it with a context. In another time, when someone else sees this, he might not know the context and believe it's the norm.
Anyway, I don't believe that in this component's case, it's really necessary due to the use you're giving to the properties. Plus, the component, being isolated by nature, won't have any conflicts with other outer scopes. One more plus, the problem with "using $scope" is actually using it explicitly, in this case you're relying on a scope (notice the "a", i.e. a conceptual scope), which will still exist in angular 2, so binding things without a namespace is ok in this circumstances.

Comment (gabrielpoca) [inline app/assets/javascripts/app/components/button/button.js]: @joaojusto bump

Comment (joaojusto) [inline app/assets/javascripts/app/components/button/button.js]: @gabrielpoca :+1:

**Pull request merged**